public/css/styles.css

```diff
@@ -120,6 +120,16 @@ td {
 .address + .address {
   margin-top: 1rem;
 }
+.inline-img {
+  height: 1.1rem;
+  margin-left: 5px;
+}
+.address-wrap {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
```

Comment:
Don't set the default `flex-wrap`. If you are using the default value then don't write it. While other person is reading it, he can think that it's overriding other CSS rule, what's not case now.